### PR TITLE
feat(hosting): update vocabulary in user modification/creation

### PR DESCRIPTION
**Pull request by lucas-chaigne**
ref: DTRSD-43496

Signed-off-by: Lucas Chaigne <lucas.chaigne@ovhcloud.com>

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/webhosting-sftp-doc`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #DTRSD-43496
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description

In webhosting manager, FTP-SSH tab, user modification and creation are updated so that user better understands options related to SFTP.

## Related

- #5988
- #5991
- #5989
- #5993

### CODE REVIEW

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -751,23 +751,23 @@
   "hosting_tab_FTP_multiftp_table_ssh_state_NONE": "Désactivé",
   "hosting_tab_FTP_multiftp_table_state_rw": "Activé",
   "hosting_tab_FTP_multiftp_table_state_off": "Désactivé",
-  "hosting_tab_FTP_multiftp_table_ssh_state_active": "Activé",
-  "hosting_tab_FTP_multiftp_table_ssh_state_sftponly": "SFTP",
-  "hosting_tab_FTP_multiftp_table_ssh_state_none": "Désactivé",
+  "hosting_tab_FTP_multiftp_table_ssh_state_active": "FTP, SFTP and SSH",
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:754]: **question:** Why using word `and` into a `Messages_fr_FR.json` file?

```suggestion
  "hosting_tab_FTP_multiftp_table_ssh_state_active": "FTP, SFTP et SSH",
```

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -751,23 +751,23 @@
...
+  "hosting_tab_FTP_multiftp_table_ssh_state_sftponly": "FTP and SFTP",
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:755]: See previous comment.

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -939,16 +939,16 @@
   "hosting_tab_FTP_configuration_user_create_step2_password_password": "Mot de passe",
   "hosting_tab_FTP_configuration_user_create_step2_password_confirmation": "Confirmation du mot de passe",
   "hosting_tab_FTP_configuration_user_create_step2_password_error": "Les mots de passe entrés ne sont pas identiques.",
-  "hosting_tab_FTP_configuration_user_create_step3_question": "Souhaitez vous réellement créer l'utilisateur FTP suivant ?",
+  "hosting_tab_FTP_configuration_user_create_step3_question": "Souhaitez vous réellement créer l'utilisateur suivant ?",
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:942]:
```suggestion
  "hosting_tab_FTP_configuration_user_create_step3_question": "Souhaitez-vous réellement créer l'utilisateur suivant ?",
```

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -975,14 +975,15 @@
...
+  "hosting_tab_FTP_configuration_user_modify_step2_question": "Souhaitez-vous réellement modifier l'utilisateur avec les valeurs suivantes ?",
   "hosting_tab_FTP_configuration_user_modify_step2_user": "Utilisateur",
   "hosting_tab_FTP_configuration_user_modify_step2_home_label": "Dossier racine",
-  "hosting_tab_FTP_configuration_user_modify_step2_state_label": "Désactiver l'utilisateur",
+  "hosting_tab_FTP_configuration_user_modify_step2_state_label": "Privilèges de l'utilisateur",
   "hosting_tab_FTP_configuration_user_modify_step2_iisRemoteRights_label": "iis Remote rights",
   "hosting_tab_FTP_configuration_user_modify_step2_webDavRights_label": "WebDav rights",
-  "hosting_tab_FTP_configuration_user_modify_fail": "Une erreur est survenue lors de la demande de modification de l'utilisateur FTP.",
-  "hosting_tab_FTP_configuration_user_modify_success": "L'utilisateur FTP va être modifié.",
+  "hosting_tab_FTP_configuration_user_modify_step2_protocols_label": "Protocoles de connection",
```
Comment (antleblanc) [inline packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:984]:
```suggestion
  "hosting_tab_FTP_configuration_user_modify_step2_protocols_label": "Protocoles de connexion",
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json (hunk 1 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:754]: Hell, thanks for the review. indeed, it doesnt make sense

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json (hunk 3 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:942]: ok

File: packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json (hunk 4 above)
Comment (lucas-chaigne) [reply packages/manager/apps/web/client/app/hosting/dashboard/translations/Messages_fr_FR.json:984]: ok

**Review by antleblanc — APPROVED** [review]

**Review by mohammed-zahaf — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Pull request merged**